### DISCUSSION TITLE
Dreame L10 ULTRA (not S version)

### DISCUSSION THREAD

**Discussion by Lookatgr3g** [Tasshack/dreame-vacuum#252] [Q&A] [answered]
hello, is it possible to make the L10 Ultra version compatible, which can currently only be installed with the proprietary Dreame Home application?

I've already searched the internet, but I haven't found anything about it, I only find guides for the L10"s" ULTRA version

Thank you!

**Comment by Tasshack** [comment, marked as answer]
Available in the Beta version.
